src/convenience/frameworks.ts

```diff
@@ -289,6 +298,23 @@ const azure: AzureAdapter = (request, context) => ({
         context.res?.send?.(401, WRONG_TOKEN_ERROR);
     },
 });
+const azureV4: AzureAdapterV4 = (request, context) => {
+    type Res = NonNullable<
+        Awaited<ReturnType<AzureAdapterV4>["handlerReturn"]>
+    >;
+    let resolveResponse: (response: Res) => void;
+    return {
+        update: Promise.resolve(request.json()) as Promise<Update>,
+        header: context.res?.headers?.[SECRET_HEADER],
```

Comment:
Here, we access `context.res`

Reply:
Oh I see, we should use `request.headers.get('content-type')` instead. We can drop the context object entirely, I think.